health-economics.csl

```diff
@@ -0,0 +1,204 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style class="in-text" xmlns="http://purl.org/net/xbiblio/csl" version="1.0" demote-non-dropping-particle="sort-only">
```

Comment:
This style needs a `default-locale`, probably for US English ("en-US"). See https://github.com/citation-style-language/styles/wiki/Style-Requirements#3---style-locale